src/reducers/systemIntakeReducer.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  GET_SYSTEM_INTAKE,
+  UPDATE_SYSTEM_INTAKE
+} from '../constants/systemIntake';
+import { SystemIntakeState } from '../types/systemIntake';
+
+const initialState: SystemIntakeState = {};
+
+function systemIntakeReducer(state = initialState, action: any): any {
+  switch (action.type) {
+    case GET_SYSTEM_INTAKE:
```

Comment:
I think we can delete this, let this case pass through to default.

Reply:
I agree. State isn't being updated, so the current state of the store can be returned.